dsgrid/config/dataset_config.py

```diff
@@ -189,6 +195,15 @@ class OneTableDataSchemaModel(DSGBaseModel):
     )
 
 
+class GrowthRateDataSchemaModel(OneTableDataSchemaModel):
+    """ data schema model for growth rate format """
+
+    load_data_column_dimension: DimensionType = Field(
+        title="load_data_column_dimension",
+        description="The data dimension for which its values are in column form (pivoted) in the load_data table.",
+    )
+
+
 class DatasetConfigModel(DSGBaseModel):
     """Represents dataset configurations."""
```

Comment:
I think there should be a new class GrowthRateDatasetConfigModel(DatasetConfigModel), because the data format for growth rates can largely be the same, its just interpretation of the data that is different--to be fully usable we need to be able to say in the config that this growth rate applies to X other dataset and aligns with that dataset's dimensions in Y way. That's additional information that a regular dataset does not need.